src/common.c

```diff
@@ -103,6 +103,11 @@ uint8_t ofi_msb(uint64_t num)
 	return msb;
 }
 
+uint8_t ofi_lsb(uint64_t num)
+{
+	return ofi_msb(num & (~(num - 1)));
+}
```

Comment:
This is not the mirror of ofi_msb().  msb returns the *position* of the bit (0-63).  lsb is returning the *value* of the bit (2^x).